tests/vmidefaults_test.go

```diff
@@ -152,7 +152,7 @@ var _ = Describe("[Serial][sig-compute]VMIDefaults", Serial, decorators.SigCompu
 				Address: &api.Address{
 					Type:     api.AddressPCI,
 					Domain:   "0x0000",
-					Bus:      "0x04",
+					Bus:      "0x07",
```

Comment:
why is the bus change required ?

Reply:
right, just read the commit msg of the last commit.

Reply:
Can I resolve it then?

Reply:
sure